app/views/posts/show.html.erb

```diff
@@ -11,4 +11,6 @@
     <div class="border mt-6">
         <%= render 'shared/name_comment', comments: @post.comments %>
     </div>
+
+    <button class="px-3 py-1 bg-blue-600 rounded text-white block mr-auto mt-4 ml-2 mb-2">Add Comment</button>
```

Comment:
- [ ] [OPTIONAL] As the previous reviewer forget to mention to that, kindly make sure when I click on this button map to view for create comment because I can't access the create comment view unless I enter the route for it.